src/Module/Api/Friendica/Activity.php

```diff
@@ -40,14 +39,14 @@
  */
 class Activity extends BaseApi
 {
-	protected function rawContent(array $request = [])
+	protected function post(array $request = [], array $post = [])
```

Comment:
What is this `$post` array for? Isn't it redundant with `$request`?

Reply:
And a possible dump question: Why aren't there parameters at the `delete`, `patch` and `put` methods?

Reply:
@MrPetovan 
you got me! At first I "just" had `$post` inside the `post()` method, than I noticed I need `$request` as well and now it's redundant!

@annando it's part of the PR that all methods are now getting them, so we don't need to fetch them inside the modules anymore, we can rely on the parameters we now have :-)

Reply:
Where do you fetch the request variable?

Reply:
I'm currently preparing a replacement PR for this one, where I split the commits for a better readability (and to avoid forgetting something because of merging/rebasing to 2021.12-rc)

There I'll add a specific commit for the request fetching